x-pack/plugins/monitoring/public/components/elasticsearch/nodes/nodes.js

```diff
@@ -137,7 +137,7 @@ const getColumns = (showCgroupMetricsElasticsearch, setupMode, clusterUuid, aler
         <AlertsStatus
           showBadge={true}
           alerts={alerts}
-          stateFilter={(state) => (state.nodeId || state.nodeUuid) === node.resolver.uuid}
+          stateFilter={(state) => (state.nodeId || state.nodeUuid) === node.resolver}
```

Comment:
Good catch, this definitely looks like it was just a bug from a previous PR. 

One thing, though: I can't tell if that filter is doing what we want or not? `(a || b) === c` isn't the same as `(a === c) || (b === c)`, do you know if we want the latter?

Reply:
Yes, the former.  In my local environment `state.nodeUuid` is always undefined.